webview/src/experiments/components/table/header/TableHeader.tsx

```diff
@@ -27,7 +27,7 @@ const possibleOrders = {
 } as const
 
```

Comment:
The column placeholders have a  "Hide Column" option, but it doesn't seem to work: 

https://user-images.githubusercontent.com/43496356/197221199-2d08ba46-5680-49ed-928e-7221079d76ef.mov